packages/ursula/example/package.json

```diff
@@ -15,21 +15,22 @@
     "@freesewing/mui-theme": "latest",
```

Comment:
FYI, changes to this file will be overwritten the next time `yarn reconfigure` is run.

Were these changes required for you to start the example app? If not, I suggest reverting this file. If so, we should look into why. Note that I am able to start the example app on your branch even without these changes.